dist/znk.infra.js

```diff
@@ -2920,11 +2920,14 @@ angular.module('znk.infra.general')
                 this.setter = function(path, newVal){
                     return $q.when(entitySetter(path,newVal));
                 };
+
+                this.entityCache = $cacheFactory('entityCache' + cacheId);
```

Comment:
It can be changed in the future if we'll need, currently the $cacheFactory is fulfilling our demands.